README.md

```diff
@@ -23,5 +23,5 @@ platform:
 Run the playbook like so:
 
 ```sh
-ansible-playbook --inventory inventory.yml playbook.yml
+ansible-playbook playbook.yml
```

Comment:
what goes in playbook.yml?

Reply:
That's a file that exists in this branch

Reply:
why is it also in gitignore then? or am i missing something

Reply:
oh whoops, will fix

Reply:
I meant to gitignore the inventory not the playbook, pushed fix

Reply:
ok i see, now my question is what goes in inventory.yml, does it make sense to include an example?

Reply:
There's an example in the README itself; mostly it's the basic inventory fields (host, user, become) plus two (for now) variables `datacenter` and `node_name` for plugging into Consul and eventually Nomad configuration.

Idk if I'll commit the actual file. I'm still mulling over what things are secrets and how those might be managed. Either way it won't be until we actually have a RHEL9 box managed with this playbook.

Reply:
seems fine to me